budgetportal/management/commands/load_departments.py

```diff
@@ -79,7 +79,10 @@ def create_dept():
                     )
 
                 try:
-                    if update:
+                    exists = Department.objects.filter(
```

Comment:
the script is not working when update is set to False but the department exists. I think we could get rid of the update flag and just check if dep exists.

Reply:
I think that's the point? that it only updates when you explicitly say to update.

Reply:
but if you do not say that and your Department exists it breaks, maybe we should just skip the iteration when Dep exists and there is no update flag?

Reply:
can you just revert this for this PR? sorry I just don't want to think about this right now - I'm not sure why you need to be running this twice and there's too much else going on for me to think about this properly

Reply:
🆗